### PR TITLE
Fix OP minebots, move them into a separate file

### PR DESCRIPTION
**Pull request by Markolie**
Fixes https://github.com/ParadiseSS13/Paradise/issues/6453
Minebots now do normal damage. Also moved minebots to a separate file.

🆑 Markolie
fix: Mine bots once again only do ten damage in pressurized areas
/🆑

### CODE REVIEW

File: code/modules/mining/minebot.dm [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+	environment_smash = 0
+	check_friendly_fire = 1
+	stop_automated_movement_when_pulled = 1
+	attacktext = "drills"
+	attack_sound = 'sound/weapons/circsawhit.ogg'
+	ranged = 1
+	sentience_type = SENTIENCE_MINEBOT
+	ranged_message = "shoots"
+	ranged_cooldown_time = 30
+	projectiletype = /obj/item/projectile/kinetic/minebot
```
Comment (Markolie) [inline code/modules/mining/minebot.dm]: This is the only line that was changed in the move.

**Comment by Fox-McCloud** [comment]
Won't this mean minebots are absolutely useless for fighting fauna? `10` damage won't be able to take down anything other than hivelord brood.

**Comment by Anticept** [comment]
If you're going to do this, add upgrades so that it can be stepped up.

Also, minebots aren't nerfed by atmosphere. They do full damage (at least they did to me earlier today during a test).

**Comment by Markolie** [comment]
Oh wait, you're right. I'm a derp and didn't consider the pressure. Stand by - don't merge this yet.

**Comment by Markolie** [comment]
Okay, this is properly fixed now and back to the old situation.

File: code/modules/projectiles/guns/energy/kinetic_accelerator.dm [hunk 2]
```diff
@@ -153,15 +153,15 @@
 		var/obj/item/weapon/gun/energy/kinetic_accelerator/KA = loc
 		KA.modify_projectile(BB)
 
-		var/turf/proj_turf = get_turf(BB)
```
Comment (Fox-McCloud) [inline code/modules/projectiles/guns/energy/kinetic_accelerator.dm]: Ooooops.

**Comment by Anticept** [comment]
Again: they need damage upgrades if you're only going to make them do 10 damage. That's horrible.

**Comment by Crazylemon64** [comment]
@Anticept They're pressure-based now. Or are you asking about station-side damage, too?

**Comment by DesolateG** [comment]
Perhaps allow the KA upgrades from the exosuit fabricator to work on minebot KA?

**Comment by Markolie** [comment]
That's quite out of scope of this pull request I'm afraid.

**Comment by Fox-McCloud** [comment]
This is just a fix PR to return minebots to their previous functionality; it wasn't intended for them to shoot 40 damage projectiles while on station (and they didn't).

**Comment by Fox-McCloud** [comment]
@Markolie 

Edited your changelog just a tiny bit; hope you don't mind.

**Comment by Markolie** [comment]
How dare you!

**Comment by Denthamos** [comment]
I like the idea of making the minebots able to accept upgrades to increase their functionality as they are finally getting fixed.  Allow them to accept the KA upgrades from the mining vendor and RnD, then remove the ranged upgrade option for the minebot.

**Pull request merged**